LayoutTests/fast/text/text-combine-text-transform.html

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<body style="writing-mode: vertical-rl">
+Test text-combine: hoizontal + text-transform<br><br>
```

Comment:
```suggestion
Test text-combine-upright: all + text-transform<br><br>
```
Can you also change the file names to text-combine-upright-text-transform.html ?

Reply:
Done both - rename of test and other.